lib/galaxy/config.py

```diff
@@ -305,7 +305,7 @@ def __init__( self, **kwargs ):
             self.use_tool_dependencies = True
         else:
             self.tool_dependency_dir = None
-            self.use_tool_dependencies = False
+            self.use_tool_dependencies = os.path.exists(self.dependency_resolvers_config_file)
```

Comment:
This setting doesn't default to the sample file:

```
dependency_resolvers_config_file=[ 'config/dependency_resolvers_conf.xml', 'dependency_resolvers_conf.xml' ],
```

So ``use_tool_dependencies`` should be False as it was before unless the admin sets up a file explicitly or am I missing somthing?